src/Open3D/Visualization/Rendering/Filament/FilamentScene.cpp

```diff
@@ -272,6 +272,8 @@ GeometryHandle FilamentScene::AddGeometry(
     RenderableManager::Builder builder(1);
     builder.boundingBox(aabb)
             .layerMask(FilamentView::kAllLayersMask, FilamentView::kMainLayer)
+            .castShadows(true)
+            .receiveShadows(true)
```

Comment:
This is enabled by default

Reply:
It doesn't seem to be enabled by default, because when it's not there, there are no shadows. (This is self-shadowing, I only have one object in the scene, e.g. cactusgarden or readingroom)